srml/staking/src/lib.rs

```diff
@@ -1144,7 +1161,8 @@ impl<T: Trait> OnFreeBalanceZero<T::AccountId> for Module<T> {
 		<Nominators<T>>::remove(stash);
 	}
 }
-
+/*
+// TODO: manage Aura reporting an offline validator somehow
```

Comment:
needs to have an issue filed

Reply:
it's yet to be done; though i'm not sure it's still relevant.

Reply:
We've walked back existing functionality in this PR, so it should be logged. Aura can pretty well tell who's online or not, as soon as we've got uncles.

Reply:
(either way, we've got a policy of "TODO must have an issue filed". so either an issue should go up or the TODO should go away)

Reply:
sure, it's not ready for merge right yet. that's exactly why i placed a check box for this in the PR's main text: 
![image](https://user-images.githubusercontent.com/138296/59379909-293cce00-8d58-11e9-8c40-d61660cfe181.png)